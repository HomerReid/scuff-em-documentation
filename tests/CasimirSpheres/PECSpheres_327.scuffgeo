OBJECT UpperSphere
	MESHFILE  Sphere_327.msh
	DISPLACED 0 0 3
ENDOBJECT

OBJECT LowerSphere
	MESHFILE Sphere_327.msh
ENDOBJECT
