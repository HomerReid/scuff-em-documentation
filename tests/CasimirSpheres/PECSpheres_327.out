# scuff-cas3D run on supersgj1 at 03/09/16::02:25:19
# data file columns: 
#1: transform tag
#2: z-force 
#3: z-force error due to numerical Xi integration 
3.0 -1.037111e-02 3.244331e-06 
3.4 -2.572386e-03 7.448846e-07 
3.8 -8.328525e-04 2.225608e-07 
4.3 -2.569085e-04 6.854971e-08 
4.9 -7.851886e-05 2.054455e-08 
5.5 -2.846554e-05 7.336618e-09 
6.2 -1.017478e-05 2.744329e-09 
7.0 -3.653039e-06 1.032218e-09 
7.8 -1.481995e-06 4.383811e-10 
8.9 -4.984971e-07 1.528827e-10 
10.0 -1.917880e-07 5.962519e-11 
12.6 -2.925206e-08 9.401546e-12 
15.8 -4.693014e-09 1.613781e-12 
20.0 -7.026292e-10 2.761232e-13 
25.0 -1.168884e-10 6.103851e-14 
33.0 -1.258139e-11 1.149691e-14 
41.0 -2.687075e-12 4.008608e-15 
50.0 -4.485456e-13 1.216965e-15 
63.0 -7.021193e-14 3.302312e-16 
79.0 -1.140880e-14 7.829406e-17 
100 -1.715157e-15 8.554785e-18 
